cns/restserver/util.go

```diff
@@ -616,6 +616,16 @@ func (service *HTTPRestService) getNetworkContainerDetails(networkContainerID st
 	return containerDetails, containerExists
 }
 
+func storeNcStatus(ncDetails containerstatus) cns.ContainerStatus {
```

Comment:
this reads like the function is storing a nc status. maybe `toCNSContainerStatus` is a more appropriate name?

Reply:
This function is deleted since the struct changed.